common/apache-poi/META-INF/MANIFEST.MF

```diff
@@ -2,18 +2,22 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Apache POI
 Bundle-SymbolicName: org.apache.poi
-Bundle-Version: 3.11.0.qualifier
-Bundle-ClassPath: target/ooxml-schemas-1.1.jar,
- target/poi-3.11.jar,
- target/poi-ooxml-3.11.jar,
- target/poi-ooxml-schemas-3.11.jar
+Bundle-Version: 3.17.0.qualifier
+Bundle-ClassPath: target/ooxml-schemas-1.3.jar,
+ target/poi-3.17.jar,
+ target/poi-ooxml-3.17.jar,
+ target/poi-ooxml-schemas-3.17.jar,
+ target/curvesapi-1.04.jar,
+ target/commons-collections4-4.1.jar
```

Comment:
I would move collections to separate bundle, so it can be reused without apache poi.